test/collectionUtils/InlineValueSetTypePredicate.test.ts

```diff
@@ -0,0 +1,91 @@
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import InlineValueSetTypePredicate from "../../src/collectionUtils/InlineValueSetTypePredicate";
+import DataType from "../../src/model/dataTypes/DataType";
+import MemberVariable from "../../src/model/dataTypes/MemberVariable";
+
+describe("InlineValueSetTypePredicate", () => {
+  let predicate: InlineValueSetTypePredicate;
+  let entityDefinitionBuilder: EntityDefinitionBuilder;
+  let entity: EntityDefinition;
+  let elementType: DataType;
+  let otherType: DataType;
+  let wrongNs: DataType;
+  let systemStringType: DataType;
+  let systemDateType: DataType;
+  let valueMember: MemberVariable;
+  let wrongNameMember: MemberVariable;
+  let wrongTypeMember: MemberVariable;
+  let wrongSystemTypeMember: MemberVariable;
+
+  beforeEach(() => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5963561)